.github/workflows/docker-build-push-ecr.yaml

```diff
@@ -2,6 +2,10 @@ name: Docker Build and Push
 
 on:
   workflow_call:
+    secrets:
+      docker_secrets:
+        description: "Comma-delimited list of secrets to pass to docker build"
```

Comment:
that what the secret should contain but not what this secret ref is. It would be more like. Github (org/repo/environment) secret pass to the docker build workflow.

Reply:
Smth like this?